vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListReferencesController.java

```diff
@@ -0,0 +1,56 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.IReference;
+import edu.asu.diging.vspace.core.model.SortByField;
+import edu.asu.diging.vspace.core.services.IReferenceManager;
+
+@Controller
+public class ListReferencesController {
+    
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private IReferenceManager referenceManager;
+
+    
+    @RequestMapping("/staff/references/list")
+    public String listSpacesWithoutNum( @RequestParam(value = "sort", required = false) String sortedBy,
```

Comment:
do you need this method? Can't you just define default values in the below method?